pkg/security/scc/byrestrictions.go

```diff
@@ -25,15 +26,24 @@ func (s ByRestrictions) Less(i, j int) bool {
 type points int
 
 const (
-	privilegedPoints points = 20
+	privilegedPoints points = 200000
 
-	hostVolumePoints       points = 10
-	nonTrivialVolumePoints points = 5
+	hostVolumePoints       points = 100000
+	nonTrivialVolumePoints points = 50000
 
-	runAsAnyUserPoints points = 4
-	runAsNonRootPoints points = 3
-	runAsRangePoints   points = 2
-	runAsUserPoints    points = 1
+	runAsAnyUserPoints points = 40000
+	runAsNonRootPoints points = 30000
+	runAsRangePoints   points = 20000
+	runAsUserPoints    points = 10000
+
+	capDefaultPoints  points = 5000
+	capAddOnePoints   points = 300
+	capAllowAllPoints points = 4000
+	capAllowOnePoints points = 10
+	capDropAllPoints  points = -3000
+	capDropOnePoints  points = -50
+	capMaxPoints      points = 10000
```

Comment:
Should we set `capMaxPoints` to 9999 so `runAsUserPoints` + `capMaxPoints` won't be equal to `runAsRangePoints`?